pkg/host/types/interfaces.go

```diff
@@ -0,0 +1,161 @@
+package types
```

Comment:
not an expert on MockGen: does it make things any worse to keep these interfaces in `pkg/host` package? Whether in host.go or in a different file for each interface

Reply:
In this case package with the interface implementation (e.g. kernel pkg) will not be able to import interface because of cyclic import.

manager (part of the host pkg) -> imports implementations
implementations -> imports interfaces

We have two options:
- move types + interfaces out of the `host` pkg
- move manager out of the `host` pkg

I decided to move types + interfaces.

Reply:
ok, thanks for elaborating that. Makes sense to leave the `types/interfaces.go` file